src/phoenix/server/app.py

```diff
@@ -230,6 +231,7 @@ def _lifespan(
     tracer_provider: Optional["TracerProvider"] = None,
     enable_prometheus: bool = False,
     clean_ups: Iterable[Callable[[], None]] = (),
+    start_up_callbacks: Iterable[Callable[[], None]] = (),
```

Comment:
nit: naming is inconsistent with the param above - either `startups` / `cleanups` or  `start_up_callbacks` / `clean_up_callbacks` would be better

Reply:
ok i'll make them them both callbacks